openmdao_book/features/building_blocks/drivers/genetic_algorithm.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>Is there a reason why we set each driver option separately rather than just passing them as args to the constructor?  Seems like passing them as args would be cleaner.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I think the reason is that they were written before we started using the constructor args or even before we were passing kwargs to the parent. I will put up a story to go through the docs and modify our examples to use constructor args for all classes where they are available.